src/Symfony/Component/Workflow/DefinitionBuilder.php

```diff
@@ -61,8 +61,6 @@ public function clear()
     }
 
     /**
-     * @param string $place
-     *
      * @return $this
      */
     public function setInitialPlace($place)
```

Comment:
Did you forget to add the type-hint here?

Reply:
:thinking: actually, this is more complicated. This param was wrong. We forgot to update it / deprecate this method / add a new one. Now we accept an array of initial places.

I forgot that in #30468 

I will open a new PR to fix this. This change is valid though

Reply:
In that case we shouldn‘t remove the `@param` annotation but replace it with a more appropriate one.

Reply:
Thanks again, here we go https://github.com/symfony/symfony/pull/32258

Reply:
Let's block this PR until the other is merged. Then I will be able to drop this method in master :)

Reply:
#32258 has been merged now

Reply:
Rebased, :+1: Thanks